vars/getJobTimeouts.groovy

```diff
@@ -2,17 +2,24 @@
 
 List<Integer> call(Map params, String region){
     // handle params which can be a json list
-    def test_config = groovy.json.JsonOutput.toJson(params.test_config)
-    def cmd = """
-    #!/bin/bash
-    export SCT_CLUSTER_BACKEND="${params.backend}"
-    export SCT_CONFIG_FILES=${test_config}
-    ./docker/env/hydra.sh output-conf -b "${params.backend}"
-    """
-    def testData = sh(script: cmd, returnStdout: true).trim()
-    println(testData)
-    testData = testData =~ /test_duration: (\d+)/
-    testDuration = testData[0][1].toInteger()
+    if (params.stress_duration == "") {
```

Comment:
in case `params.stress_duration` is not defined in pipeline at all (we have other pipelines that use this function, if not all of them), I think this might be considered `false` and fail in else on integer conversion. May I be right?